applications/oci-transaction-lifecycle.md

```diff
@@ -127,27 +125,27 @@ If you've already started implementing your project or it is part of a larger re
 | 0c. | Unit Tests | Core functions will be fully covered by unit tests to ensure functionality and robustness. |
-| 0d. | Integration Tests | Integration tests will be developed including new and modified tests |
-| 1. | EOSIO Transaction Resource Cost Estimation | Nodes will provide an estimate of resources (central processing unit (CPU), random access memory (RAM), and internet bandwidth (NET)) to perform a transaction when a transaction is sent to compute_transaction.  The transaction will be applied subjectively to the local chain to calculate resource costs like CPU and NET and also determine the resulting deltas in RAM usage. The response will be returned to the client to inform them of the costs associated with the given transaction data. |  
-| 2. | EOSIO Subjective Billing Improvements | Nodes will provide an added leeway per account to allow a more permissive “subjective billing” rate (rate of utilization) to provide a better user experience while maintaining system integrity. The existing subjective CPU decay window will be made configurable from the existing hard-coded value of 24 hours to allow node opperators to provide a more relaxed subjective CPU penalty to their users. The existing `disable-subjective-account-billing` will be expanded to also apply to the 3-strike rule allowing block producers to prevent abnormal transaction loss for accounts expected to sometimes fail during production. |  
+| 0d. | Integration Tests | Integration tests will be developed, including new and modified tests. |
+| 1. | EOSIO Transaction Resource Cost Estimation | Nodes will provide an estimate of resources (central processing unit (CPU), random access memory (RAM), and internet bandwidth (NET)) to perform a transaction when a transaction is sent to compute_transaction. The transaction will be applied subjectively to the local chain to calculate resource costs like CPU and NET and determine the resulting deltas in RAM usage. The response will be returned to the client to inform them of the costs associated with the given transaction data. |  
+| 2. | EOSIO Subjective Billing Improvements | Nodes will provide an added leeway per account that allows a more permissive “subjective billing” rate (rate of utilization) in order to provide a better user experience while maintaining system integrity. The existing subjective CPU decay window will be made configurable from the existing hard-coded value of 24 hours to allow node operators to provide a more relaxed subjective CPU penalty to their users. The existing `disable-subjective-account-billing` function will be expanded to also apply to the 3-strike rule, allowing block producers to prevent abnormal transaction loss for accounts expected to sometimes fail during production. |  
 
 ### Milestone 3 - Project Completion
 
 - **Estimated Duration:** 1 week
 - **FTE:**  2
 - **Costs:** $80,000 USD
 
```

Comment:
Should there be a brief explanation of what completion activities occur during this week?